src/data/navigation/sections/events.js

```diff
@@ -39,6 +39,10 @@ module.exports = [
     title: "API reference",
     path: "/events/api.md",
   },
+  {
+    title: "Consume events",
```

Comment:
We could create a nested menu, to add a new section `Consume Events Examples` and for each example, we can add a separate page to a child page. I think we will have more than one example in the documentation.

Reply:
Makes sense. Added a subsection for examples and a separate page for the current runtime action example that is now linked to in the original `consume-events.md` page